jib-core/src/test/java/com/google/cloud/tools/jib/blob/BlobTest.java

```diff
@@ -77,6 +108,7 @@ private void verifyBlobWriteTo(String expected, Blob blob) throws IOException {
     byte[] expectedBytes = expected.getBytes(StandardCharsets.UTF_8);
     Assert.assertEquals(expectedBytes.length, blobDescriptor.getSize());
 
+    @SuppressWarnings("resource") // no leak
```

Comment:
We can actually change the mocked `OutputStream` to just `Bytestreams.nullOutputStream()` to avoid the suppression.